va/va.h

```diff
@@ -3799,8 +3837,10 @@ typedef  struct _VACodedBufferSegment  {
      */
     void               *next;
 
+    VALookAheadInfo   lpla_info;
+
     /** \brief Reserved bytes for future use, must be zero */
-    uint32_t                va_reserved[VA_PADDING_LOW];
+    //uint32_t                va_reserved[VA_PADDING_LOW];
```

Comment:
the structure of VALookAheadInfo is five DWORD, so , this commit change the size of this structure.

Reply:
same question, is the VALookAheadInfo size exactly the same as va_reserved?